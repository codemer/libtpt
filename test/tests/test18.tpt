@# This is the comment test.  This line shouldn't even show up
	@# This offset comment should generate a blank line with no space.
Comment test.	@# All the space leading up to this comment should be ignored.

@set(x, 100)	@# set my variable.\
@if (x &&		@# check my variable,
	1 < 2) {		@# and another condition.
Expression was true.
}
