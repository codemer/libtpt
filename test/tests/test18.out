
Comment test.

Expression was true.
